e2e/interchain_accounts_test.go

```diff
@@ -40,6 +42,44 @@ func (s *InterchainAccountsTestSuite) RegisterInterchainAccount(ctx context.Cont
 	return err
 }
 
+// QueryIncentivizedPacketsForChannel queries the incentivized packets on the specified channel.
+func (s *InterchainAccountsTestSuite) QueryIncentivizedPacketsForChannel(
```

Comment:
I think we should put all incentivized tests under one test suite. Since ics29 was added in v4 but ics27 was added in v3. We will want to do base ics27 tests using v3 for cross compatibility purposes, but incentivization tests should only occur as far back as v4